docs/src/increase-observability/metrics/_index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Monitor metrics
+weight: 5
+description: See all of the live infrastructure metrics available to give you an overview of resource usage.
+aliases:
+  - /dedicated/architecture/metrics.html
+---
+
+Platform.sh projects are accompanied by live infrastructure metrics that provide an overview of resource usage for environments.  
+
+Within the console, metrics can be found for an environment under **Metrics**.
+
+The information under **Metrics** shows usage metrics for:
+
+[Dedicated environments](../../dedicated/overview/_index.md):
+each of the three hosts in your [N+1 configuration](../../dedicated/architecture/_index.md) and their average.
+
+![A screenshot of what the metrics dashboard displays for Dedicated environments](/images/metrics/all-dedicated.png "0.45")
+
+Grid environments: for the average of your service and app containers.
+
+![A screenshot of what the metrics dashboard displays for Grid environments](/images/metrics/all-grid.png "0.45")
+
```

Comment:
### Currently unsupported regions

If your project is hosted on regions US-1 and EU-1, please note that your project will not avail of Infrastructure metrics. 

These regions are running Legacy components and that do not support Infrastructure Metrics. There is an effort underway by Platform.sh to upgrade these regions to newer components directly. This will not require any action on customer side unless specifically communicated within the upcoming months. 

Contact our [Support team](https://console.platform.sh/-/users/~/tickets/open) if you have any questions.